backend/src/main/dto/InvoiceDTO.ts

```diff
@@ -1,13 +1,19 @@
 export interface InvoiceCreationDTO {
   totalAmount: number;
-  quantity: number;
+  paymentType: paymentType;
   date: Date;
   description: string;
   productionId: number;
 }
 export interface InvoiceUpdateDTO {
   totalAmount?: number;
-  quantity?: number;
+  paymentType?: paymentType;
   date?: Date;
   description?: string;
 }
+
+export enum paymentType {
+  PROGRESS = 'PROGRESS',
```

Comment:
I believe IN_PROGRESS would be less confusing than PROGRESS. Thoughts?

Reply:
Stakeholders told us to put "progress" according to the meeting notes.

Reply:
Yeah I think "in progress" would be better, and I think "deposit" should be "initial deposit" but ask the stakeholders before making the change.

Reply:
I think we should follow the stakeholders meeting notes for now and ask them later.